code/modules/mob/living/carbon/human/human.dm

```diff
@@ -690,6 +690,10 @@
 	else if (href_list["lookitem"])
 		var/obj/item/I = locate(href_list["lookitem"])
 		usr.examination(I)
+	else if (href_list["listitems"])
+		var/obj/item/I = locate(href_list["listitems"])
```

Comment:
Honestly I just think it should be trader trenchcoat specific.

Any reason why this href is on mobs and not the trenchcoat itself?